workers/External/src/startup.cc

```diff
@@ -65,7 +65,8 @@ std::string get_random_characters(size_t count) {
 
 // Entry point
 int main(int argc, char** argv) {
-    srand(time(nullptr));
+    auto now = std::chrono::high_resolution_clock::now();
+    srand(std::chrono::time_point_cast<std::chrono::nanoseconds>(now).time_since_epoch().count());
```

Comment:
(Also, where's the `#include` for `rand` and `srand`? Probably want `#include <cstdlib>` and use `std::srand`, `std::rand`.)